vmp/helper/tf.go

```diff
@@ -0,0 +1,36 @@
+package helper
+
+// InterfaceToStringArray converts a interface{} whose underlying type is []string. Useful for parsing Terraform resource values.
+func InterfaceToStringArray(attr interface{}) []string {
```

Comment:
Maybe ToStringArray because we know input is an interface.

Reply:
Yeah that's originally what I called it, but Go doesn't have method overloading. So if we want another "ToStringArray" func in the future then we cannot use that name.